qa/suites/deepsea/tier2/drive-replacement/1-deepsea.yaml

```diff
@@ -6,12 +6,15 @@ tasks:
                         target: '*'
                         data_devices:
                             all: true
-                            limit: 4
+                            limit: 99 # this is altered with dynamic value before stage.2
```

Comment:
How about to use: negative valumes for the limit here to use as number of reserved disk?